### PR TITLE
BTFS-2426: fix adding files failed issue; name changes.

### PR DESCRIPTION
**Pull request by laipogo**

### CODE REVIEW

File: src/bundles/config.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import memoize from 'p-memoize'
 import toUri from 'multiaddr-to-uri'
 import { createAsyncResourceBundle, createSelector } from 'redux-bundler'
 
-const DEFAULT_URI = 'https://ipfs.io'
+const DEFAULT_URI = 'https://gateway.btfs.io:8080'
```
Comment (taiyangc) [inline src/bundles/config.js:5]: Not on 8080, just https://gateway.btfs.io

File: patches/ipld-explorer-components+1.2.1.patch [hunk 2]
```diff
@@ -22,7 +22,7 @@ index 2872093..c6c1436 100644
        }) ? React.createElement(
          'a',
 -        { className: 'link avenir ml2 pa2 fw5 f6 blue', href: 'https://ipfs.io/ipfs/' + cid, target: '_external' },
-+        { className: 'link avenir ml2 pa2 fw5 f6 blue', href: 'https://gateway.btfssoter.io/btfs/' + cid, target: '_external' },
```
Comment (taiyangc) [inline patches/ipld-explorer-components+1.2.1.patch:25]: remove soter

File: src/files/modals/share-modal/ShareModal.stories.js [hunk 3]
```diff
@@ -10,6 +10,8 @@ storiesOf('Files/Modals', module)
     <div className='ma3'>
       <ShareModal
         onLeave={action('Leave')}
-        link='https://ipfs.io/ipfs/QmQK3p7MmycDutWkWAzJ4hNN1YBKK9bLTDz9jTtkWf16wC' />
+        //TODO: upload the materials to btfs and replace
+        //link='https://ipfs.io/ipfs/QmQK3p7MmycDutWkWAzJ4hNN1YBKK9bLTDz9jTtkWf16wC' />
+        link='https://gatewaybtfs.io/btfs/QmQK3p7MmycDutWkWAzJ4hNN1YBKK9bLTDz9jTtkWf16wC' />
```
Comment (taiyangc) [inline src/files/modals/share-modal/ShareModal.stories.js:15]: gateway.btfs.io

File: patches/ipld-explorer-components+1.2.1.patch (hunk 2 above)
Comment (laipogo) [reply patches/ipld-explorer-components+1.2.1.patch:25]: it's removed actually. 
looks like it's a bug of github

**Review by taiyangc — APPROVED** [review]

**Pull request merged**